editor/code_editor.cpp

```diff
@@ -1541,6 +1541,7 @@ void CodeTextEditor::_toggle_scripts_pressed() {
 void CodeTextEditor::_error_pressed(const Ref<InputEvent> &p_event) {
 	Ref<InputEventMouseButton> mb = p_event;
 	if (mb.is_valid() && mb->is_pressed() && mb->get_button_index() == BUTTON_LEFT) {
+		goto_error();
```

Comment:
That makes emitting the `error_pressed` signal useless, no? Since you removed the only connection to it, which was then supposed to call `goto_error()`.

Reply:
Yeah, its not needed anymore